style.css

```diff
@@ -226,6 +227,171 @@ header nav {
   display: block;
 }
 
+/* modal */
+
+.modal-container {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  position: fixed;
+  background-color: #f7f7f9;
+  top: 0;
+  left: 50%;
+  transform: translate(50%, 50%) scale(0.1);
+  visibility: hidden;
+  transition: transform 0.5s, top 0.5s;
+}
+
+.modal-container .card {
+  margin: 0;
+  background: #fff;
+  padding: 0;
+  height: 100%;
+  border-radius: 15px;
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.2);
+}
+
+.modal-container.open-popup {
+  top: 55%;
+  transform: translate(-50%, -50%) scale(0.8);
+  visibility: visible;
+  width: 100%;
+  z-index: 99;
+}
```

Comment:
Good job with the details popup, your popup for mobile is perfect, but as mentioned by the previous reviewer https://github.com/belaymit/Portfolio-microverse/pull/6/files#r1024390290 Please fix the height of the modal popup since once Opened the close button for the popup is out of the screen and we are not able to close the popup 
see below :point_down: 
![image](https://user-images.githubusercontent.com/97521557/202279702-3178e3e1-1f13-43e6-9787-a78cce7d96a6.png)

Maybe you can try to increase the top % or fix the scale() property you can learn more about [modal popup here](https://www.w3schools.com/howto/howto_css_modals.asp)

Reply:
I do not know the problem but it is working on mine!
![image](https://user-images.githubusercontent.com/27350247/202281595-dcd31f27-56ba-4d77-88df-46f9cbc4d4ab.png)
![image](https://user-images.githubusercontent.com/27350247/202281747-ecb0cdd5-a30d-436c-9840-cc43f43deda4.png)

Reply:
![image](https://user-images.githubusercontent.com/27350247/202289867-5c9ae46b-0fe1-41ac-9b41-273963c38f59.png)